tests/app/test_retriever.py

```diff
@@ -28,28 +28,29 @@ def setUp(self):
             database.db.create_all()
             self.db = database.db
 
-    def populate_database(self, x=0, convo=False, draft=False, multiple_users=False):
+    def populate_database(self, x=0, single=True, convo=False, draft=False, multiple_users=False):
         with self.engine.connect() as con:
             for i in range(x):
-                msg_id = str(uuid.uuid4())
-                year = 2016
-                month = random.choice(range(1, 13))
-                day = random.choice(range(1, 29))
-                sent_date = datetime(year, month, day)
-                query = 'INSERT INTO secure_message(msg_id, subject, body, thread_id, sent_date, read_date,' \
-                        ' collection_case, reporting_unit, survey) VALUES ("{0}", "test","test","", ' \
-                        '"{1}", "2017-02-03 00:00:00", "ACollectionCase", "AReportingUnit", ' \
-                        '"SurveyType")'.format(msg_id, sent_date)
-                con.execute(query)
-                query = 'INSERT INTO status(label, msg_id, actor) VALUES("SENT", "{0}", "respondent.21345")'.format(
-                    msg_id)
-                con.execute(query)
-                query = 'INSERT INTO status(label, msg_id, actor) VALUES("INBOX", "{0}", "SurveyType")'.format(
-                    msg_id)
-                con.execute(query)
-                query = 'INSERT INTO status(label, msg_id, actor) VALUES("UNREAD", "{0}", "SurveyType")'.format(
-                    msg_id)
-                con.execute(query)
+                if single:
```

Comment:
I am not sure why populate database is one big function . For discussion- Would it be easier to follow if we had seperate functions such as populate_database_single_user, populate_draft etc